### PR TITLE
mbid-mapping-writer

### PR DESCRIPTION
**Pull request by mayhem**
Not much new to say that wasn't already said about this project.

One question remains for me, however: Do I need to add tests for this? The matching logic has tests, so the most crucial component of this is being tested. The rest is about coordinating listens to be looked up and is heavily monitored.

I'm not sure we gain much by adding tests at this point....

### CODE REVIEW

**Review by alastair — COMMENTED** [review]
Looks pretty good to me. A few small comments.

Regarding testing, I think you're right - specific tests on the actual matching code is good, but we shouldn't go crazy trying to do testing of everything. If you're happy with the amount of testing then I think that's good enough. If we find bugs in the future that could benefit from being tested then we could add it then.

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -106,3 +106,12 @@ services:
       # where we put the compiled output to be served by the service "web"
       - ../listenbrainz/webserver/static:/static:z
 
+  mbid_mapping_writer:
```
[the remark this answers is not in the document]
Comment (mayhem) [reply docker/docker-compose.yml]: Good point -- no longer needed, so removed.

File: consul_config.py.ctmpl [hunk 2]
```diff
@@ -60,7 +60,7 @@ TIMESCALE_ADMIN_LB_URI = "SERVICEDOESNOTEXIST_timescale-listenbrainz"
 
 {{if service "pgbouncer-williams"}}
 {{with index (service "pgbouncer-williams") 0}}
-DB_CONNECT_MAPPING = "dbname=musicbrainz_json_dump user=musicbrainz host={{.Address}} port={{.Port}} password=musicbrainz"
+MBID_MAPPING_DATABASE_URI = "dbname=musicbrainz_json_dump user=musicbrainz host={{.Address}} port={{.Port}} password=musicbrainz"
 {{end}}
 {{else}}
 DB_CONNECT_MAPPING = "SERVICEDOESNOTEXIST_pgbouncer-williams"
```
[the remark this answers is not in the document]
Comment (mayhem) [reply consul_config.py.ctmpl]: Oh, good catch!

**Review by alastair — APPROVED** [review]
let's try it!

**Pull request merged**